contracts/StablecoinConverter.sol

```diff
@@ -169,6 +180,16 @@ contract StablecoinConverter is EpochTokenLocker {
       * @param volumes executed buy amounts for each order identified by index of owner-orderId arrays
       * @param prices list of prices for touched tokens indexed by next parameter
       * @param tokenIdsForPrice price[i] is the price for the token with tokenID tokenIdsForPrice[i]
+      *
+      * Requirements:
+      * 1. Solutions for this `batchIndex` are currently being accepted.
+      * 2. feeToken price is specified.
+      * 3. `tokenIdsForPrice` is sorted.
+      * 4. Number of touched orders does not exceed `MAX_TOUCHED_ORDERS`.
+      * 5. Each touched order is valid at current `batchIndex`.
+      * 6. `executedSellAmount` does not exceed order's remaining amount.
+      * 7. Limit Price of each touched order is respected.
+      * 8.
```

Comment:
What's

<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia3.giphy.com%2Fmedia%2F2pVNVd922Igak%2Fgiphy-downsized-medium.gif&token=eight" data-gitmeme-token="eight"><img src="https://media3.giphy.com/media/2pVNVd922Igak/giphy-downsized-medium.gif" title="Created by gitme.me with /eight"/></a>

Should we just stick to 
- bullet
- points
- ?

Reply:
Ya sure... I'll fix this soon